src/libmongoc/src/mongoc/mongoc-cursor.c

```diff
@@ -845,8 +842,7 @@ _mongoc_cursor_monitor_failed (mongoc_cursor_t *cursor,
    /* we sent OP_QUERY/OP_GETMORE, fake a reply to find/getMore command:
     * {ok: 0}
     */
-   bson_init (&reply);
-   bson_append_int32 (&reply, "ok", 2, 0);
+   bsonBuildDecl (reply, kv ("ok", int32 (0)));
```

Comment:
```suggestion
   bsonBuildDecl (reply, kvl ("ok", 2, int32 (0)));
```
Suggest preserving length to avoid call to `strlen`.

Reply:
Resolving in response to https://github.com/mongodb/mongo-c-driver/pull/1094#discussion_r1004799519.